Sources/StreamChat/Utils/Logger/Logger.swift

```diff
@@ -121,25 +121,37 @@ public enum LogConfig {
         }
     }
     
+    /// Destination types this logger will use.
+    ///
+    /// Logger will initialize the destinations with its own parameters. If you want full control on the parameters, use `destinations` directly,
+    /// where you can pass parameters to destination initializers yourself.
+    public static var destinationTypes: [LogDestination.Type] = [ConsoleLogDestination.self] {
+        didSet {
+            invalidateLogger()
```

Comment:
shouldn't this be called in `willSet` closure instead?

Reply:
Based on our discussion outside of GitHub -> `invalidateLogger` will recreate the Logger with new destination type(s).